x/mint/keeper/genesis.go

```diff
@@ -18,6 +18,15 @@ func (k Keeper) InitGenesis(ctx sdk.Context, data *types.GenesisState) {
 	k.SetMinter(ctx, data.Minter)
 	k.SetParams(ctx, data.Params)
 
+	// developer vesting module account delta initialization.
+	if err := k.SetTruncationDelta(ctx, types.DeveloperVestingModuleAcctName, sdk.ZeroDec()); err != nil {
```

Comment:
do we not need to add these in export genesis?

Reply:
Added